docs/advanced/pipeline.rst

```diff
@@ -0,0 +1,81 @@
+.. currentmodule:: psycopg
+
+.. _pipeline-mode:
+
+Pipeline mode support
+=====================
+
+.. versionadded:: 3.1
+
+The *pipeline mode* allows PostgreSQL client applications to send a query
+without having to read the result of the previously sent query. Taking
+advantage of the pipeline mode, a client will wait less for the server, since
+multiple queries/results can be sent/received in a single network transaction.
```

Comment:
What do we mean by 'network transaction' here? 'network packet'? 'network interaction'?

Might be good to avoid the term 'transaction', since that has a very specific (and different) meaning in databases.

Reply:
Yes, I think the same. I lifted a few of those paragraphs directly from PostgreSQL documentation (first paragraph, [here](https://www.postgresql.org/docs/14/libpq-pipeline-mode.html)), so I'd have assumed it very strictly nitpicked. But it sounds strange to me too.

I would call it network roundtrip.